src/components/TokenGetter/TokenSelect.tsx

```diff
@@ -1,13 +1,13 @@
 import { useTokensContext } from '../../contexts/tokens'
 import { useEffect, useMemo, useRef, useState } from 'react'
 import { Loader } from '../common/Loader'
-import stub from '../../assets/no_symbol.svg'
-import { ReactComponent as DdIcon } from '../../assets/dropdown.svg'
+import stub from '../../assets/no_symbol.svg?url'
```

Comment:
I see many different ways to import modules, some with relative paths, some with a tsconfig file (`@/`). 
Let's go through the whole project and make it look the same?